tests/commons/glob/test_glob.py

```
@@ -0,0 +1,100 @@
+    ("./sub/file.txt", "*.xml", False),
+])
+def test_pathMatchesGlob(path, glob, expected):
+    assert pathMatchesGlob(TEST_FOLDER / "data/glob" / path, glob) == expected
+
+
+def test_visitGlobPaths():
+    assert visitGlobPaths
+    assert GlobMatchVisitor
+
+    class TestVisitor(GlobMatchVisitor):
+        def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)